src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ExecutionPlanGraphUtils.cs

```diff
@@ -100,6 +102,61 @@ public static ExecutionPlanEdges ConvertShowPlanEdgeToExecutionPlanEdge(Edge edg
             };
         }
 
+        public static Dictionary<string, string?> GetRowMetrics(PropertyDescriptorCollection properties)
+        {
```

Comment:
Have you gotten these properties reviewed from @erinstellato-ms? 
Please switch it to cost metrics object here.

Reply:
Are these the ONLY metrics we are looking at to determine what is most expensive?  If so, we do want cost also.

Reply:
Properties like cost and subtree cost were already present. The properties added here are ones that were missing.